### DISCUSSION TITLE
✨ MegaLinter package

### DISCUSSION THREAD

**Discussion by nvuillam** [dagger/dagger#12057] [Ideas]
### What are you trying to do?

I'd like to create a reusable dagger package to run [MegaLinter](https://oxsecurity.github.io/megalinter/latest/)

Would such PR be accepted ?
If yes, where in dagger sources could I implement it ?

### Why is this important to you?

It could allow dagger users to easily integrate MegaLinter within their dagger pipelines :)

### How are you currently working around this?

I could just do docker run but i'd prefer to have a reusable item :)

**Comment by slumbering** [comment]
hey @nvuillam 👋 
That's a great idea. Sorry for the late answer 🤭
Dagger is now using SDKs, in which language would you prefer to create this reusable package ?

**Comment by nvuillam** [comment]
MegaLinter is in python, with a little bit of node.js, would it fit ?
Are there some guidelines to build on dagger something equivalent to a docker based github action ?
